api/app/signals/apps/reporting/models/export.py

```diff
@@ -2,9 +2,10 @@
 # Copyright (C) 2019 - 2021 Gemeente Amsterdam
 from django.contrib.gis.db import models
 
-from signals.apps.reporting.utils import _get_storage_backend
+# TODO:: Verify that Horeca is unused and maby delete?
```

Comment:
As far as I am aware this is not used anymore. There is no task that triggers this export configured in Jenkins.
Double check with Jelle and the VNG and if not used it can be removed.

Reply:
This was used in 2019 to create special reports for "horeca" nuisance complaints around ADE. We have since deactivated those special reports but were asked to keep te the code around in case similar questions were to crop up in the future. There will be an ADE 2022, so it is probably smart to reach out pre-emptively.